hw04/BarbieWalletBalance.cs

```diff
@@ -14,21 +14,66 @@ public class BarbieBank : MonoBehaviour
     Have the function return a new generic array with the correct parameters. 
     */
 
+    //answer:
     void Start()
     {
+        //question 2
+        BarbieWorld<string> barbiehat = new BarbieWorld<string>();
+
         int[] biggerWallet = BarbieWallet(500, 600);
```

Comment:
This problem is very close. Needs an open and close. When we call this function we want to pass in a function. We want to pass in some type - here it is an `int`.

```
    void Start()
    {
        int[] biggerWallet = BarbieWallet<int>(500, 600);
        Debug.Log(biggerWallet.Length + " " + biggerWallet[0] + " " + biggerWallet[1]);

        GetMoney<int>(500, 600);
    }
```